docs/glossary.rst

```diff
@@ -35,13 +39,15 @@ Glossary
       An abstract entity, representing a single |ContractType| or
       |ContractInstance|.
 
+   Contract Interface
+      Refers to the collection of methods through which one may interact with a contract.
+
    Contract Instance
       A contract account with its own address, storage, and balance,
       representing a single deployment of a particular |ContractType|.
 
    Contract Type
-      Refers to a specific contract or library, able to deployed to a particular
-      |Network|
+      Refers to a specific contract or library, able to be deployed to a particular |Network|
```

Comment:
Perhaps including a little more context would be helpful to the reader. For example: `Refers to a specific contract or library, able to be deployed to a particular |Network| as a |Contract Instance|`

This helps me realize that `Contract Type` is a Smart Contract class|source that has no deployed attributes.

Reply:
This is actually not relevant anymore, and will be removed.